src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -72,12 +116,17 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
 const getString = (param: string | string[]): string =>
   Array.isArray(param) ? param[0] : param;
 
-const getQueryVariablesFromUrlSearch = (patchId: string, search: string) => {
+const getQueryVariablesFromUrlSearch = (
+  patchId: string,
+  search: string,
+  page: number
```

Comment:
nit: page isn't actually from the url search params and this function isn't accessing url search params directly. we could probably change the name of the function to exclude `UrlSearch`